src/components/LeadersItem/LeadersItem.tsx

```diff
@@ -11,7 +11,7 @@ type LeadersItemProps = {
 
 const LeadersItem: FC<LeadersItemProps> = ({leaders}: LeadersItemProps) => (
   <div className={styles.leadersItem}>
-    {leaders &&
+    {leaders.length > 0 &&
```

Comment:
no need to write '> 0'. You can just check leaders.length, because 0 is a false and when leaders.length is 0,  this (leaders.length > 0) and this (leaders.length) checks will return false. 

But to avoid rendering '0' you need to write !!leaders.length